lib/platform/gitlab/index.ts

```diff
@@ -100,28 +100,27 @@ export async function initRepo({
     }
     config.defaultBranch = res.body.default_branch;
     config.baseBranch = config.defaultBranch;
+    if (res.body.http_url_to_repo === null) {
+      throw new Error('no http_url_to_repo found');
```

Comment:
Is it possible we can instead just fall back to the "old" functionality if this field isn't found? Otherwise, I need to do a lot of due diligence before I can merge - checking if this field is always present, checking about what release of GitLab it was added in, etc. If instead we just gracefully fall back in its absence - i.e. functionality that apparently satisfies the majority of users - then it's much lower risk and easier to merge.

Reply:
Sure, will try that.